### ISSUE TITLE
Invalid address

### ISSUE THREAD

**Issue by Johnsavadkuhi** [binance-chain/javascript-sdk#104]
Hello everyone  

this issue is related to javascript_SDK and java_SDK if  https://testnet.binance.org  use javascript_sdk .

i use this private key  to get address : 90335b9d2153ad1a9799a3ccc070bd64b4164e9642ee1dd48053c33f9a3a05e9

but returned address in java_sdk and https://testnet.binance.org  is differ 
address from JAVA_SDK :  bnc1hgm0p7khfk85zpz5v0j8wnej3a90w7098fpxyh  
address from testnet.binanace.org : tbnb1hgm0p7khfk85zpz5v0j8wnej3a90w709zzlffd

i know bnc or tbnb  is hrp  and there is  1hgm0p7khfk85zpz5v0j8wnej3a90w709  in both

**Comment by Johnsavadkuhi** [comment]
I think it is because of encoding and bech32

**Comment by wally-yu** [comment]
haven't yet tried java sdk, but Javascript SDK's result is identical with test-net.
i guess the incorrect one is because of "prefix" while encoding with bech32?

**Comment by notatestuser** [comment]
> haven't yet tried java sdk, but Javascript SDK's result is identical with test-net.
> i guess the incorrect one is because of "prefix" while encoding with bech32?

This is correct - the prefixes are different.

**Comment by ahsannawaz11** [comment]
bnbClient.chooseNetwork("testnet") // or this can be "mainnet"
bnbClient.setPrivateKey(privKey)

on this line
bnbClient.setPrivateKey(privKey) facing error. which is "unable to query the address on the blockchain. try sending it some funds first: tbnb1dusl7s4khzl2ta4peptmj2tmz0m4zg008crvun". What is this? can you please help me. i'm using java script sdk for testnet

**Comment by bheemtl** [comment]
Hello everyone

I am facing an issue related to javascript_SDK on bnb-chain/javascript-SDK.

I used private key(d45278ad9968e47429c1844c4145788b767082fa36b8a7dc7a3201fa242393b5) to generated the testnet address and got the below address 
tbnb1uzn7fntldky6tfl86ayhra383xv7thrhaw8key and generated ERC20 address with 0x prefix: 0x962f4bdf209677c009c479d79808db6bf72f255f

It's a valid address according to https://testnet.bscscan.com and also working for coin transfer but when I tried to send a coin using testnet address with the javascript-sdk using the below code

bnbClient.chooseNetwork("testnet")
bnbClient.setPrivateKey(privKey)

Then it showing error: which is "unable to query the address on the blockchain. try sending it some funds first: tbnb1uzn7fntldky6tfl86ayhra383xv7thrhaw8key". 

Can any one help me to understand this? I am using java script sdk for the testnet.

Thanks
